builtin/settingtypes.txt

```diff
@@ -110,9 +110,9 @@ doubletap_jump (Double tap jump for fly) bool false
 #    enabled.
 always_fly_fast (Always fly and fast) bool true
 
-#    The time in seconds it takes between repeated right clicks when holding the right
-#    mouse button.
-repeat_rightclick_time (Rightclick repetition interval) float 0.25
+#    The time in seconds it takes between repeated block placements when holding
+#    the place block button.
+repeat_place_time (Place repetition interval) float 0.25
```

Comment:
This would break existing clients in a non-violent way. i.e. `repeat_rightclick_interval` won't work anymore, but they could easily fix this by renaming it to `repeat_place_time`. Thoughts?

Reply:
This needs attending to as well. I doubt this is a popular setting, but I'm pretty sure a good bunch of users have tweaked this to their requirements. IMO, the new name actually better describes the purpose of the setting, even if `place` is hardcoded to RMB.

Reply:
Backwards compatibility would be nice to have.
1) Read old setting value (if it exists)
2) Set new value
3) Remove old setting line